### PR TITLE
Update basics.sh

### PR DESCRIPTION
**Pull request by Stochastic-Squirrel**
- Removed some cask installs
- Updated brew cask install to brew install --cask
- Exports SYSTEM_VERSION_COMPAT=1 to avoid compatibility issues of various software with the major macOS update

### CODE REVIEW

File: basics.sh [hunk 1]
```diff
@@ -64,26 +64,25 @@
...
+        brew install --cask the-unarchiver
+        brew install --cask pycharm-ce
+        brew install --cask drawio
         pipx ensurepath && source ~/.zshrc
-        # Uncomment block if you want to install the microsoft stuff
-        echo "\n-> installing microsoft stuff from homebrew casks:"
-        echo "   microsoft-edge microsoft-office microsoft-teams"
-        brew cask install microsoft-edge
-        brew cask install microsoft-teams
-        brew cask install drawio
```
Comment (aj-cloete) [inline basics.sh:86]: Giving MS the axe!

File: basics.sh [hunk 2]
```diff
@@ -101,17 +100,6 @@
...
-    echo "\nIf you don't yet know magnet for OSX, you're wasting time resizing windows!"
-    echo "Here's the link: https://apps.apple.com/gb/app/magnet/id441258766?mt=12"
-    read -p "Do you want to open that link now? [y/n]" -t 5 REPLY
-    case $REPLY in
-        [yY])
-            open "https://apps.apple.com/gb/app/magnet/id441258766?mt=12"
-        ;;
-        *)
-        ;;
-    esac
```
Comment (aj-cloete) [inline basics.sh:114]: You really should be using magnet.  Everyone should be using magnet

**Review by aj-cloete — APPROVED** [review]

**Comment by aj-cloete** [comment]
I'd like to know where your tests are?

File: basics.sh (hunk 2 above)
Comment (Stochastic-Squirrel) [reply basics.sh:114]: Agreed! However i think that requiring user input via the terminal might spook some people who aren't technically orientated who might need to run this setup script

**Comment by Stochastic-Squirrel** [comment]
Tests are being dealt with in another branch. Will run the check on this PR once i have validated the Action

File: basics.sh (hunk 2 above)
Comment (aj-cloete) [reply basics.sh:114]: It times out after 5 seconds.

**Pull request merged**